.dockerignore

```diff
@@ -0,0 +1,26 @@
+**/.classpath
+**/.dockerignore
+**/.env
+**/.git
+**/.gitignore
+**/.project
+**/.settings
+**/.toolstarget
+**/.vs
+**/.vscode
+**/*.*proj.user
+**/*.dbmdl
+**/*.jfm
+**/azds.yaml
+**/bin
+**/charts
+**/docker-compose*
+**/Dockerfile*
+**/node_modules
```

Comment:
🤔 I'm not seeing any `package.json`, is this folder usually in the context? Or the `.dockerignore` was generated with some tool?

Reply:
Yeas, it was autogenerated by VS Code.